docs/standard/simd.md

```diff
@@ -0,0 +1,104 @@
+---
+title: SIMD in .NET
+description: This tutorial demonstrates how to use SIMD in C#.
+author: FIVIL
+#ms.author: [MICROSOFT ALIAS OF INTERNAL OWNER]
+ms.date: 07/05/2019
+#ms.topic: [TOPIC TYPE]
+#ms.prod: [PRODUCT VALUE]
+ms.technology: dotnet-standard
+---
+
+# Overview
+
+SIMD (Single instruction, multiple data) provides the functionality to do multiple processes on a single core to achieve peak performance. In .NET there is set of SIMD-enabled types under <xref:System.Numerics> namespace. SIMD operations can be parallelized at the hardware level. That increases the throughput of the vectorized computations, which are common in mathematical, scientific, and graphics apps.
```

Comment:
It is probably worthwhile to callout both the `System.Numerics` types and the System.Runtime.Intrinsics namespaces. 
The former provides SIMD accelerated functions (in `Vector2/3/4` and other types) and more direct SIMD access (`Vector<T>`) which may have a software fallback on some platforms (such as `arm32`).

The latter provides direct hardware access and no software fallback; so support must be explicitly queried and fallbacks need to be provided by the developer.

Reply:
Actually, I have just started to work with new `System.Runtime.Intrinsic` in .net core 3 and I will complete these documents based on this [blog post](https://devblogs.microsoft.com/dotnet/hardware-intrinsics-in-net-core/) of yours, very soon